src/helpers/tests.js

```diff
@@ -1,49 +1,27 @@
 import React from 'react';
-import ReactTestRenderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
 import 'jest-styled-components';
-import { render } from 'enzyme';
+
 import DefaultTheme from 'themes/Default';
 
 export const snapshotWithElementChildren = (TestedElement, props) => {
-  const result = ReactTestRenderer.create(
+  const result = shallow(
     <DefaultTheme>
       <TestedElement {...props}>
         <div />
       </TestedElement>
     </DefaultTheme>
-  ).toJSON();
+  );
 
-  expect(result).toMatchSnapshot();
+  expect(result.childAt(0)).toMatchSnapshot();
```

Comment:
Can't we snapshot the whole result ?
With `shallow`, isn't it supposed to be the same ?

Reply:
Without that, the `ThemeProvider` is included in the snapshot, with its `theme` prop...

It is huge *and* we don't test `ThemeProvider` after all, just its child.